Makefile

```diff
@@ -0,0 +1,211 @@
+BASE_PACKAGES := utils db settings visualization
+
+MODELS := biocro clm45 dalec ed gday jules linkages \
+	      lpjguess maat maespa preles sipnet
+
+
+MODULES := allometry assim.batch assim.sequential benchmark \
+	       data.atmosphere data.hydrology data.land \
+	       data.mining data.remote emulator meta.analysis \
+	       photosynthesis priors rtm uncertainty
+
+.PHONY = all
+
+all: ${BASE_PACKAGES} ${MODELS} ${MODULES}
+
+#### DEPENDENCY MAPPING ####
+
+${MODELS}: utils data.atmosphere
+
+#### BASE PACKAGES ####
+
+utils: .install.utils
```

Comment:
curious why do you have a utils that depends on .install.utils, etc.
would it make sense to add a dependency here that is all the R packages needed for that module and make sure they are installed. We should be able to pull them from the Dependencies file.